common/test/acceptance/pages/lms/problem.py

```diff
@@ -68,6 +74,24 @@ def is_correct(self):
         """
         return self.q(css="div.problem div.capa_inputtype.textline div.correct span.status").is_present()
 
+    def simpleprob_is_correct(self):
+        """
+        Is there a "partially correct" status showing? Works with simple problem types.
```

Comment:
I'm wondering if these simpleprob_is_* methods need to have updated docstrings. The descriptions are the same, but I'm guessing from the names they are different (for good reasons)... Could you update these to more clearly draw the distinctions between them?